app/src/main/AndroidManifest.xml

```diff
@@ -46,9 +46,11 @@
 
         <activity
             android:name="com.google.android.gms.oss.licenses.OssLicensesMenuActivity"
+            android:theme="@style/Theme.AppCompat"
```

Comment:
Wow, good find.

Reply:
Do you know why this works?

Reply:
Nope.  I launched the activity under a few different circumstances and could occasionally see other content behind it.  Transparent background sounded like a theming issue, so I thought this might fix it.  No idea beyond that.